### ISSUE TITLE
Files Not Found in celeba_wild_preprocess.py

### ISSUE THREAD

**Issue by zhumanli** [xingzhehe/AutoLink-Self-supervised-Learning-of-Human-Skeletons-and-Object-Outlines-by-Linking-Keypoints#1]
Hi, 

Thank you very much for making such great work public!

I couldn't find list_imsize_celeba.txt and celebA_training.txt from the [CelebA Dataset website ](https://mmlab.ie.cuhk.edu.hk/projects/CelebA.html), can you provide more instructions on how to obtain these two files? Thank you very much for your help.

**Comment by xingzhehe** [comment]
Hi,

Thanks for you interests in our work.

1. celebA_training.txt can be found in https://github.com/xingzhehe/GANSeg/tree/main/data/celeba_wild_raw/MAFL.

2. Very interestingly, I cannot recall where I downloaded list_imsize_celeba.txt. Maybe I generated it by myself. Anyway I have changed the preprocessing file to work without this file by reading directly from the image.

If you have any further questions, don't hesitate to reach me.

Thanks,
Xingzhe
